src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1634,6 +1634,50 @@ class EltwiseRefExecutor : public Eltwise::IEltwiseExecutor {
             });
             return;
         }
+        if (_opData.algo == Algorithm::EltwisePowerDynamic) {
```

Comment:
I would expect addition of PowerStatic optimized code, as long as this algorithm much more common.

Reply:
@dmitry-gorokhov corrected

Reply:
So the next step is to enable Conversion to PowerStatic for ARM platform: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/transformations/cpu_opset/convert_to_cpu_specific_opset.hpp#L39 (by making this transformation common). After that you will be able to remove optimized code block for PowerDynamic.
Also please peform benchmark launch on whole set of models to make sure PowerStatic transormation enablement doesn't introduce any regressions.